grc/blocks/parameter.xml

```diff
@@ -79,6 +79,23 @@ none#slurp
 part#slurp
 #end if</hide>
 	</param>
+	<param>
+		<name>Display</name>
+		<key>hide</key>
+		<value></value>
+		<type>enum</type>
+		<hide>part</hide>
+		<option>
+			<name>Show</name>
+			<key>none</key>
+			<opt>hide:none</opt>
```

Comment:
Those keys will be written in the XML file. So, those are necessary. But I am skeptical about `<opt>` tags.